types/module/module.go

```diff
@@ -286,7 +286,6 @@ func (m *Manager) SetOrderBeginBlockers(moduleNames ...string) {
 
 // SetOrderMidBlockers sets the order of set mid-blocker calls
 func (m *Manager) SetOrderMidBlockers(moduleNames ...string) {
-	// TODO: do we need this assertion? maybe its ok to not have it to only have some modules have MidBlock implementations
 	// m.assertNoForgottenModules("SetOrderMidBlockers", moduleNames)
```

Comment:
we can also remove this line since we dont need to have midblockers for all modules (which is what the TODO was referring to)